lib/cfg/cfg_Base.ml

```diff
@@ -0,0 +1,105 @@
+open Cfg_Node
+
+module CFG = struct
+  (* Auxiliary module for providing Hashtbl of a pretty printing function *)
+  module Hashtbl = struct
+    include Hashtbl
+
+    let pp pp_key pp_value ppf values =
+      Hashtbl.iter (fun key data -> Format.fprintf ppf "@[<1>%a: %a@]@." pp_key key pp_value data) values
+  end
+
+  type 'a item = {
```

Comment:
please add a shorter description of what an item is and of each field, even if it derivable from the context